in-payments.js

```diff
@@ -0,0 +1,25 @@
+const paymentsIn = [

```

Comment:
please rename file to `payments-in.js`
and the other one you can make as `payments-out.js`